src/tests/Interop/common/ComHelpers.h

```diff
@@ -50,20 +52,25 @@ namespace Internal
         return __QueryInterfaceImpl(riid, ppvObject, remain...);
     }
 }
+ #endif  
 
 // Implementation of IUnknown operations
 class UnknownImpl
 {
-public:
-    UnknownImpl() = default;
+public:  
+    UnknownImpl() : _refCount{ 1 } {};
     virtual ~UnknownImpl() = default;
 
     UnknownImpl(const UnknownImpl&) = delete;
     UnknownImpl& operator=(const UnknownImpl&) = delete;
 
-    UnknownImpl(UnknownImpl&&) = default;
+    UnknownImpl(UnknownImpl&&)  : _refCount{ 1 } {};
```

Comment:
Is this needed? If so, I am not sure this is correct because it is a "move constructor" so resetting the value to `1` seems wrong.

Reply:
Ok, I think you're right